CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
+
 # Change log
 
 ## master (unreleased)
+- PR [#143](https://github.com/palkan/logidze/pull/143) Add `:transactional` option to `#with_meta` and `#with_responsible` ([@oleg-kiviljov][])
```

Comment:
One more thing: please, add the value for your nickname's shortcut to the end of this file.